Tests/LibWeb/Text/expected/wpt-import/css/geometry/DOMMatrix-001.txt

```diff
@@ -23,7 +23,7 @@ Pass	new DOMMatrix(sequence) 6 elements
 Pass	new DOMMatrix("scale(2) translateX(5px) translateY(5px)")
 Pass	new DOMMatrix("scale(2, 2) translateX(5px) translateY(5px)")
 Pass	new DOMMatrix("scale(2)translateX(5px)translateY(5px)")
-Pass	new DOMMatrix("scale(2) translateX(calc(2 * 2.5px)) translateY(5px)")
+Fail	new DOMMatrix("scale(2) translateX(calc(2 * 2.5px)) translateY(5px)")
```

Comment:
Looks like the DOMMatrix regressions are because we end up calling `Transformation::to_matrix()` without a `PaintableBox`, so we don't attempt to resolve the `calc()` even though it just contains a single px value. That's unfortunate. I can't think of a straightforward way of solving that though, so it can wait.